app/helpers.php

```diff
@@ -170,17 +178,21 @@ function get_signed_language_for_written_language(string $code): string
      */
     function get_language_exonym(string $code, string $locale = '', bool $capitalize = true): null|string
     {
-        if ($locale === '') {
-            $locale = locale();
-        }
+        $locale = $locale === '' ? locale() : $locale;
+
+        $locale = match ($locale) {
+            'asl' => 'en',
+            'lsq' => 'fr',
+            default => $locale
+        };
 
         switch ($code) {
-            case 'fcs':
+            case 'lsq':
                 return trans('locales.'.$code, [], $locale);
```

Comment:
`lsq` doesn't support the `capitalized` option. The localization currently has it capitalized by default. So you could probably do the reverse and lowercase this if the capitalize argument is false. Although it would be more robust to handle both lower and uppsercasing so we aren't dependent on how it is written in the translation file.